website/public/js/controllers/groupsCtrl.js

```diff
@@ -173,6 +173,7 @@ habitrpg.controller("GroupsCtrl", ['$scope', '$rootScope', 'Shared', 'Groups', '
           member.petCount = Shared.countPets($rootScope.countExists(member.items.pets), member.items.pets);
           member.mountCount = Shared.countMounts($rootScope.countExists(member.items.mounts), member.items.mounts);
         $scope.profile = member;
+        $scope.e = member.items.gear.equipped;
```

Comment:
If you're referring to the renaming, I can do that right now, but if you're referring to more extensive refactoring, it would probably be better if you do that, since I'm not 100% sure what you're looking for.